router.go

```diff
@@ -2,14 +2,15 @@ package main
 
 import (
 	"fmt"
-	"github.com/alphagov/router/handlers"
-	"github.com/alphagov/router/logger"
-	"github.com/alphagov/router/triemux"
-	"labix.org/v2/mgo"
 	"net/http"
 	"net/url"
 	"sync"
 	"time"
+
+	"github.com/alphagov/router/handlers"
```

Comment:
They are, but within groups.  There's an emerging convention to put the imports into groups, with all the stdlib imports first, and all the 3rd party imports separately.

Additionally, the [goimports](https://godoc.org/golang.org/x/tools/cmd/goimports) tool automatically arranges them this way.